src/htmldrawer.js

```diff
@@ -85,11 +85,15 @@ class HTMLDrawer extends OpenSeadragon.DrawerBase{
             };
         }
 
+        // In theory, HTML drawer should cope well with canvas node type too,
```

Comment:
This seems fine... I don't think anyone is using canvas tiles with the HTML drawer. What do you think? I assume it was at least possible before.

Is it still possible if you explicitly give it canvas objects? It just doesn't automatically convert them? Just trying to understand the implications.

Reply:
I honestly don't know and I am even inclined to the fact the HTML drawer worked only with images. This change will force the system to always convert to an image element. So yes, it automatically converts any non-image element.

Reply:
Seems reasonable. If someone needs to use true canvases with the HTML drawer it'll be pretty niche and it would be good for us to communicate with them to learn exactly what their needs are. Meanwhile, let's stick with just images.

Reply:
HIstorically, html used only images. Looked into old implementation - it called `getImage()` on the tile method `drawHTML(...)`. There was no mechanism to define multiple formats supported by a drawer like now. So the canvas extension was just an optimization on my side, but honestly, nobody is probably using the HTML drawer anyway so if tests are satisfied, so am I :D

Reply:
I believe there have still been people using the HMTL drawer in rare cases where the canvas and WebGL drawers don't work (some mobile scenarios where there are bugs in the browser, I think). Last I heard of that it was a while ago, though, so hopefully even they have been able to switch to the more modern drawers.